### PR TITLE
[parse] Added fullText function to Parse.Query

### PR DESCRIPTION
**Pull request by ChrisGrant**
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://parseplatform.org/Parse-SDK-JS/api/2.1.0/Parse.Query.html#fullText
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Comment by ChrisGrant** [comment]
@wesleygrimes my reasoning was that it is only ever used inside of query. It can't be used anywhere else as far as I can see. I can make it top level if you think that's better though?

**Comment by wesleygrimes** [comment]
As long as it’ll never be used anywhere else I’m good with it as is. Thanks!

**Review by wesleygrimes — APPROVED** [review]

File: types/parse/index.d.ts [hunk 1]
```diff
@@ -675,6 +676,13 @@ declare namespace Parse {
             // Sort documentation https://docs.mongodb.com/v3.2/reference/operator/aggregation/sort/#pipe._S_sort
             sort?: {[key: string]: 1|-1};
         }
+
+        // According to https://parseplatform.org/Parse-SDK-JS/api/2.1.0/Parse.Query.html#fullText
+        interface FullTextOptions {
+          language:	string;
```
Comment (amcasey) [inline types/parse/index.d.ts]: Are these all required?  Frequently, options types have optional members.

Comment (amcasey) [reply types/parse/index.d.ts]: Ping @ChrisGrant

Comment (weswigham) [reply types/parse/index.d.ts]: The docs agree; one of the examples is `query.fullText('field', 'term', { language: 'es', diacriticSensitive: true });`.  @ChrisGrant you wanna add ` ?` to the end of each of these props for correctness?

**Review by weswigham — CHANGES_REQUESTED** [review]

**Comment by ChrisGrant** [comment]
@weswigham Apologies for the delay, been away for a week. You're right! Done now.

**Review by weswigham — APPROVED** [review]

**Pull request merged**